conans/test/functional/generators/virtualenv_python_test.py

```diff
@@ -37,18 +36,17 @@ def package_info(self):
         client.run("create . ")
         client.save({"conanfile.txt": base}, clean_first=True)
         client.run("install . -g virtualenv_python")
-        name = "activate_run_python.sh" if platform.system() != "Windows" else "activate_run_python.bat"
-        contents = load(os.path.join(client.current_folder, name))
-        self.assertNotIn("OTHER", contents)
-        self.assertIn("PATH=", contents)
-        self.assertIn("LD_LIBRARY_PATH=", contents)
-        self.assertIn("DYLD_LIBRARY_PATH=", contents)
 
         if platform.system() != "Windows":
-
+            contents = load(os.path.join(client.current_folder, "environment_run_python.sh.env"))
```

Comment:
``client.load("environment_run_python.sh.env")``

Reply:
There are many `load(os.path.join(client.current_folder` in the codebase, let's leave it for a dedicated PR

Reply:
Sure, there are like 40 remaining, I already fixed like a couple of hundreds in another PR. We don't need to fix them in this PR, but new tests would benefit from simpler syntax, I see no reason why not using it already in this PR

Reply:
I'll merge and create engineering issue.